task/minio-cli/0.1/minio-cli.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: minio-cli
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "minio cli"
+spec:
+  description: >-
+   MinIO is a High Performance Object Storage released under Apache License v2.0.
+   It is API compatible with Amazon S3 cloud storage service.
+
+   This task lists all the buckets available on the MinIO server.
+
+  params:
+  - name: server-url
+    type: string
+    description: MinIO server URL
+  - name: server-alias
+    type: string
+    description: MinIO server alias
+  - name: key-tokens
+    type: string
+    description: Secret for the MinIO server
+  - name: cli-command
+    description: The minio cli command to tun
+    type: string
+    default: "--help"
+  steps:
+  - name: run-command
+    image: docker.io/minio/mc:RELEASE.2020-10-03T02-54-56Z@sha256:19c0ad83449d760bf85ea81f6794d1223639858acb423c095fe34b54cbfacd12
+    script: |
+      mc config host add $(params.server-alias) $(params.server-url) $minio_access_key $minio_secret_key
+      mc $(params.cli-command) $(params.server-alias)
```

Comment:
How can I run the following command `mc mb play/mybucket` or `mc mb s3/mybucket --region=us-west-1` using the above script. Any guide would be very helpful :slightly_smiling_face: